README.md

```diff
@@ -1 +1,99 @@
-# space_travellers_hub
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Space Travelers Hub
+
+## Description the project.
+> Bookstore - An app that allows you to display a list of added books, add new book and remove a book, built using React, Redux
+
+## Screenshots:
+
+![screenshot](./src/assets/rockets.PNG)
+
+![screenshot](./src/assets/missions.PNG)
+
+![screenshot](./src/assets/myProfile.PNG)
+
+
+## Built With
+
```

Comment:
- [ ] As Mentioned by the previous reviewer, kindly use the new [Readme ](https://github.com/microverseinc/readme-template) template and fill in all the sections 🙏

Reply:
Thank you for the feedback.
Issue resolved